concourse/pipelines/create-deployer.yml

```diff
@@ -413,6 +413,7 @@ jobs:
     serial: true
     plan:
     - get: paas-cf
+      passed: [vpc]
```

Comment:
I would even pin it to concourse-prepare-deploy because it's the last job in the pipeline that requires paas-cf, and I want to run expunge on the same version. What do you think?